docs/guides/projects.md

```diff
@@ -1,32 +1,136 @@
 # Working on projects
 
+uv can manage the development of a Python project from the ground up.
+
 ## Creating a new project
 
+You can create a new Python project using the `uv init` command:
+
+```console
+$ uv init hello-world
+$ cd hello-world
+```
+
+Alternatively, you can initialize a project in the working directory:
+
+```console
+$ cd hello-world
+$ uv init
+```
+
+This will create the following directory structure:
+
 ```
-uv init
+.
+├── .venv
+├── pyproject.toml
+├── README.md
+├── uv.lock
+└── src
+    └── hello-world
+        └── __init__.py
 ```
 
 ### Working on an existing project
 
-<!-- What makes a project compatible with uv commands? What are my alternatives? -->
+If your project already contains a standard `pyproject.toml`, you can start
+using uv without any extra work. Commands like `uv add` and `uv run` will
+create the lockfile and virtual environment the first time they are run.
+
+If you are migrating from an alternative Python package manager, you may need to
+edit your `pyproject.toml` manually to use uv. uv uses the `[tool.uv]` section
+of the `pyproject.toml` to support non-standard features, such as development
+dependencies. Alternative Python package managers may use different sections,
+or a custom format altogether.
+
+## Project structure
+
+A project consists of a few important parts that work together and allow uv to
+manage your project.
+
+### `pyproject.toml`
+
+The `pyproject.toml` contains metadata about your project:
+
+```toml
+[project]
+name = "hello-world"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+dependencies = []
+
+[tool.uv]
+dev-dependencies = []
+```
+
+This is where you specify dependencies, as well as details about the project
+such as it's description or license. You can edit this file manually, or use
+commands like `uv add` and `uv remove` to manage your project through the
+CLI.
+
+### `.venv`
+
+The `.venv` folder contains your project's virtual environment, a Python
+environment that is isolated from the rest of your system. This is where uv will
+install your project's dependencies.
 
+### `uv.lock`
+
+`uv.lock` is a lockfile that contains exact information about your project's
```

Comment:
Are we recommending to check this in to VCS (does that depend on application vs. library)?

Reply:
I think it should always be checked into VCS